sparse/impl/KokkosSparse_spiluk_numeric_impl.hpp

```diff
@@ -167,25 +153,17 @@ struct IlukWrap {
     }
 
     // divide. lhs /= rhs
-    KOKKOS_INLINE_FUNCTION
-    void divide(scalar_t &lhs, const scalar_t &rhs) const { lhs /= rhs; }
-
     KOKKOS_INLINE_FUNCTION
     void divide(const member_type &team, scalar_t &lhs,
                 const scalar_t &rhs) const {
       Kokkos::single(Kokkos::PerTeam(team), [&]() { lhs /= rhs; });
       team.team_barrier();
     }
 
-    // add. lhs += rhs
+    // gemm, alpha is hardcoded to -1, beta hardcoded to 1
     KOKKOS_INLINE_FUNCTION
-    void add(scalar_t &lhs, const scalar_t &rhs) const { lhs += rhs; }
-
-    // multiply: return (alpha * lhs) * rhs
-    KOKKOS_INLINE_FUNCTION
-    scalar_t multiply(const scalar_t &alpha, const scalar_t &lhs,
-                      const scalar_t &rhs, scalar_t *) const {
-      return alpha * lhs * rhs;
+    void gemm(const scalar_t &A, const scalar_t &B, scalar_t &C) const {
```

Comment:
Why do you call it ```gemm``` as A, B, C are scalars? I just feel it might be confusing. 
Can we call it multiply–accumulate (MAC) or multiply-add (MAD)?

Reply:
Done